src/build_hpo.sh

```diff
@@ -45,21 +46,31 @@ function check_err() {
 # Remove any previous images of hpo
 function cleanup() {
 	echo -n "Cleanup any previous kruize images..."
-	docker stop hpo >/dev/null 2>/dev/null
+	eval "${CONTAINER_RUNTIME} stop hpo >/dev/null 2>/dev/null"
 	sleep 5
-	docker rmi $(docker images | grep hpo | awk '{ print $3 }') >/dev/null 2>/dev/null
-	docker rmi $(docker images | grep hpo | awk '{ printf "%s:%s\n", $1, $2 }') >/dev/null 2>/dev/null
+	eval "${CONTAINER_RUNTIME} rmi $(${CONTAINER_RUNTIME} images | grep hpo | awk '{ print $3 }') >/dev/null 2>/dev/null"
+	eval "${CONTAINER_RUNTIME} rmi $(${CONTAINER_RUNTIME} images | grep hpo | awk '{ printf "%s:%s\n", $1, $2 }') >/dev/null 2>/dev/null"
 	echo "done"
 }
 
 function set_tags() {
-	HPO_REPO=$(echo ${HPO_DOCKER_IMAGE} | awk -F":" '{ print $1 }')
-	DOCKER_TAG=$(echo ${HPO_DOCKER_IMAGE} | awk -F":" '{ print $2 }')
+	HPO_REPO=$(echo ${HPO_CONTAINER_IMAGE} | awk -F":" '{ print $1 }')
+	DOCKER_TAG=$(echo ${HPO_CONTAINER_IMAGE} | awk -F":" '{ print $2 }')
 	if [ -z "${DOCKER_TAG}" ]; then
 		DOCKER_TAG="latest"
 	fi
 }
 
+function resolve_container_runtime() {
+	IFS='=' read -r -a dockerDeamonState <<< $(systemctl show --property ActiveState docker)
+	[[ "${dockerDeamonState[1]}" == "inactive" ]] && CONTAINER_RUNTIME="podman"
+	if ! command -v podman &> /dev/null
+	then
```

Comment:
Minor nit, we use `if cond; then` in the same line everywhere else so for consistency we can follow the same here. LGTM otherwise!

Reply:
I have cleaned up the syntax, squashed and pushed a new commit